doc/backing-store-crd.md

```diff
@@ -16,172 +22,206 @@ Adding support for a new type of backing-store is rather easy as it requires jus
+*In the YAML, the value of `storage` should adhere to the [K8s Memory Resource Units format](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-memory).
 ```shell
-noobaa -n noobaa backingstore create pv-pool bs --num-volumes 3 --pv-size-gb 32 --storage-class STORAGE-CLASS-NAME
+noobaa -n noobaa backingstore create pv-pool <BACKINGSTORE NAME> --num-volumes <> --pv-size-gb <INT >= 17> --storage-class <THE SC THAT WILL BE USED FOR PV PROVISIONING>
 ```
 ```yaml
 apiVersion: noobaa.io/v1alpha1
 kind: BackingStore
 metadata:
   finalizers:
   - noobaa.io/finalizer
-  labels:
-    app: noobaa
-  name: bs
-  namespace: noobaa
+  name: <>
+  namespace: <>
 spec:
   pvPool:
-    numVolumes: 3
+    numVolumes: <>
     resources:
       requests:
-        storage: 32Gi
+        storage: <SEE NOTE ABOVE*>
     storageClass: STORAGE-CLASS-NAME
   type: pv-pool
 ```
 
+## Examples
+AWS S3
+```shell
+noobaa -n app-namespace backingstore create aws-s3 aws-backingstore --access-key AKIAIOSFODNN7EXAMPLE --secret-key wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY --target-bucket personal-bucket
+```
+```yaml
+apiVersion: noobaa.io/v1alpha1
+kind: BackingStore
+metadata:
+  finalizers:
+  - noobaa.io/finalizer
+  name: aws-backingstore
+  namespace: app-namespace
+spec:
+  awsS3:
+    secret:
+      name: user-created-aws-s3-secret
+      namespace: secret-namespace
+    targetBucket: personal-bucket
+  type: aws-s3
+```
 
-#### Credentials change
+Persistent Volume Pool
+```shell
+noobaa -n noobaa backingstore create pv-pool pv-backingstore --num-volumes 3 --pv-size-gb 17 --storage-class local-path
+```
+```yaml
+apiVersion: noobaa.io/v1alpha1
+kind: BackingStore
+metadata:
+  finalizers:
+  - noobaa.io/finalizer
+  name: pv-backingstore
+  namespace: app-namespace
+spec:
+  pvPool:
+    numVolumes: 3
+    resources:
+      requests:
+        storage: 17Gi
+    storageClass: local-path
+  type: pv-pool
+```
 
-In case the credentials of a backing-store need to be updated due to a periodic security policy or concern, the appropriate secret should be updated by the user, and the operator will be responsible for watching changes in those secrets and propagating the new credential update to the NooBaa system server.
 
+## Modifying a Backing Store's Credentials
+If a user wishes to change a backing store's credentials, the appropriate secret should be updated by the user, and the operator will propagate the new credentials to the system server.
 
-# Read Status
+A backing store's secret can be found in the CR's YAML's `spec` field, under a key named `secret`.
 
-Here is an example of healthy status (see below an example of non-healthy status):
+# Resource Status
+Below is an example of a healthy backing store's status:
 
 ```yaml
```

Comment:
I think there were some updates here: all the fields are capitalized and there is also a mode section, for example:

```
    Last Heartbeat Time:   2023-03-16T11:32:16Z
    Last Transition Time:  2023-03-16T11:32:16Z
    Message:               BackingStorePhaseReady
    Reason:                Backing store mode: OPTIMAL
    Status:                True
    Type:                  Upgradeable
  Mode:
    Mode Code:   OPTIMAL
    Time Stamp:  2023-03-16 05:34:36.425471879 +0000 UTC m=+1530.291704034
```